sdk/sdk-ids/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "solana-sdk-ids"
```

Comment:
I went back and forth on other possible names for the crate, but `solana-sdk-ids` seems like the clearest